kinto/core/views/batch.py

```diff
@@ -67,13 +68,43 @@ class BatchRequest(colander.MappingSchema):
     body = BatchPayloadSchema()
 
 
+class BatchResponseSchema(colander.MappingSchema):
+    status = colander.SchemaNode(colander.Integer())
+    path = colander.SchemaNode(colander.String())
+    headers = colander.SchemaNode(colander.Mapping(unknown='preserve'),
+                                  validator=string_values,
+                                  missing=colander.drop)
+    body = colander.SchemaNode(colander.Mapping(unknown='preserve'),
+                               missing=colander.drop)
+
+
+class BatchResponseBodySchema(colander.MappingSchema):
+    responses = colander.SequenceSchema(BatchResponseSchema(missing=colander.drop))
+
+
+class BatchResponse(colander.MappingSchema):
+    body = BatchResponseBodySchema()
+
+
+class ErrorResponseSchema(colander.MappingSchema):
+    body = ErrorSchema()
+
+
+batch_responses = {
+    '200': BatchResponse(description='Return a list of operation responses.'),
+    '400': ErrorResponseSchema(description='The request was badly formatted.'),
+    'default': ErrorResponseSchema(description='an unknown error occurred.')
+}
+
 batch = Service(name="batch", path='/batch',
                 description="Batch operations")
 
 
 @batch.post(schema=BatchRequest,
             validators=(colander_validator,),
-            permission=NO_PERMISSION_REQUIRED)
+            permission=NO_PERMISSION_REQUIRED,
+            tags=['Batch'], operation_id='batch',
```

Comment:
Tags could be defined on the service, but cornice swagger doesn't support it yet (There's an issue for that https://github.com/Cornices/cornice.ext.swagger/issues/61). Operation ids need to be isolated by method/verb.